plugins/CoreAdminHome/templates/optOut.twig

```diff
@@ -69,7 +69,7 @@
 
     {% if not showConfirmOnly %}
     {% set loadInNewWindow = isSafari and trackVisits %}
-    <form method="post" action="?module=CoreAdminHome&amp;action=optOut{% if language %}&amp;language={{ language }}{% endif %}{% if loadInNewWindow %}&amp;setCookieInNewWindow=1{% endif %}" {% if loadInNewWindow %}target="_blank"{% endif %}>
+    <form method="post" action="?{{ queryParameters }}" {% if loadInNewWindow %}target="_blank"{% endif %}>
```

Comment:
Update: I would actually try to use `{{ queryParameters|url_encode }}` where `queryParameters` should be an array. This will actually call `http_build_query`: http://us2.php.net/manual/en/function.http-build-query.php . The parameter in `$this->queryParameters` can be removed then and calling `Url::getQueryStringFromParameters()` should be not needed as well